Sources/Common/Background Queue/Task Data/DeletePushNotificationQueueTaskData.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+public struct DeletePushNotificationQueueTaskData: Codable {
```

Comment:
Lots of files in this PR are structs that git says are new files. 

They are not new, they are moved and I modified them from `internal` to `public` so code in tracking module could access them.